tests/processors/merge_trials_identifiers/test_processor.py

```diff
@@ -31,7 +29,6 @@ def test_updates_trial_with_records_identifiers(self, conn, trial, record):
         updated_trial = conn['database']['trials'].find_one(id=trial)
 
         assert updated_trial['identifiers'] == record_attrs['identifiers']
-        assert updated_trial['updated_at'].replace(tzinfo=None) > trial_last_updated
```

Comment:
This was removed because the triggers that update `updated_at` were not cloned into the test database. If we want them in the test db we have write and execute them again in here if we keep the SQLAlchemy-based schema dump or make the dump with `pg_dump`. Considering that we rarely test updates and when we do we can manually update the `updated_at` field, I figured it might not worth the effort.
@vitorbaptista  What do you think?